cmac/default_config.py

```diff
@@ -360,8 +360,8 @@
     # CACTI C-SAPR 2 field names.
     'cacti_csapr2_ppi': {
         # Radar field names
-        'input_zdr': 'differential_reflectivity',
-        'reflectivity': 'uncorrected_reflectivity_h',  # need to change to input_reflectivity
+        'input_zdr': 'attenuation_corrected_differential_reflectivity',
+        'reflectivity': 'attenuation_corrected_reflectivity_h',  # need to change to input_reflectivity
```

Comment:
nope.. should not be the attenuation corrected

Reply:
Is there just a corrected_differential_reflectivity?

Reply:
@scollis Not that I see, i posted the level b fields below